plugins/dynamic-plugins-info/src/components/DynamicPluginsInfoContent/DynamicPluginsInfoContent.tsx

```diff
@@ -7,7 +7,11 @@ import { DynamicPluginsTable } from '../DynamicPluginsTable/DynamicPluginsTable'
 export const DynamicPluginsInfoContent = () => (
   <>
     <ContentHeader title="">
-      <SupportButton title="Support">Some placeholder text</SupportButton>
+      <SupportButton>
+        The administration area in Developer Hub enables administrators to
```

Comment:
This button only shows up within the context of the list of installed plugins, but the description is probably more appropriate for the entire administration section.  I think probably we need a more focused text string here that's just for the table view that this plugin provides, @ShiranHi @invincibleJai WDYT?

Reply:
@ciiay For now, let's use a string like "All of the installed plugins", this will be consistent with similar text in the app like in the catalog page or API page.